### DISCUSSION TITLE
OMI Media Sync 08/15/23

### DISCUSSION THREAD

**Discussion by jimmy6DOF** [omigroup/media#66] [General]
## OMI Media Sync 08/15/23

A new meetup to see whats remaining from the old list and to add any new items 👍 

TODO "Metaverse Festival / Burn"  -- Page on Website (@indiebio & @MTV969) 
TODO "Twitter Keys" and posting protocol agreement with Jimmy6dof

DISCUSSION POINTS:
- Metaverse Festival  Press release / Media Calendar [ 1 item a month : e.g .REddit AMA etc ] 
-

**Comment by indiebio** [comment]
Media Calendar:
Proposed idea:
Once a month have an event, e.g. content creation workshop or how to create a vehicle. So , for example:

This month, on this date, jin will show how to create a vehicle in hyperfy, using a free model from sketchfab.

1. This is scheduled, and then advertised on Twitter, Reddit, Discord, Mastodon etc etc.
2. On the day and time, he has a live demo in hyperfy, and it is recorded.
3. We then make that available on Youtube.
4. We then share that link on  Twitter, Reddit, Discord, Mastodon etc etc.
5. Then we have a AMA on reddit as a "debrief" about the event.
6. Repeat the next month with a different thing.

**Reply by indiebio** [reply]
So this is an easy, low commitment way to get involved and /or see what we are doing, and it also links to building the actual festival in Oct 2024.

**Reply by indiebio** [reply]
These meetups, and the campfires, should happen in different virtual worlds, not always the same.

**Reply by indiebio** [reply]
details here from previous hackmd: https://hackmd.io/@indiebio/metaverse-float-parade

**Reply by indiebio** [reply]
related, Mrmetaverse did a "how to do a booth" video that we could repeat in a live event as one of these events. https://www.youtube.com/watch?v=6bchlTeC0P4

**Comment by indiebio** [comment]
Have a virtual gettogether with the people of the virtual burns, to share lessons learnt, a campfire event. 
TODO: 

1. Find a specific person/people who are part of the virtual burns to talk to.
2. Set a date with them to have a virtual world campfire event.
3. Choose a virtual world that works for them.
4. Advertise it
5. Then do the list, have it, record it, make it available on youtube, have a debrief reddit AMA... do the content things.

**Reply by indiebio** [reply]
jin says having an expressive world would be better, i.e. where the avatars can use their hands, express themselves. Hyperfy is not really good enough for that.

**Reply by indiebio** [reply]
Mention 2020 multiverse initiative

**Comment by indiebio** [comment]
Another campfire is getting this overlap, get the crypto and VR people around a campfire
![image](https://github.com/omigroup/media/assets/23474368/60c8a2fa-027a-4895-8dda-4e54d42c35dd)

**Comment by jimmy6DOF** [comment]
Consideration for :
- Academic / GeoSpatial / Photogrammetry Theme 
- Burner (BRCVR Et Al) Campfire loose planning sessions

**Reply by indiebio** [reply]
e.g. https://www.youtube.com/watch?v=esXD6LDl4cY

**Reply by indiebio** [reply]
So have a event that shows how to bring in real world into a virtual world. Can demo google earth, cesium, terria.js etc. 
This could become a similar type of festival event for real world stuff, digital twins, urban design, see https://discord.com/channels/770382203782692945/1110526662266855607 but for now let's not lose focus and stay with the artful playful burner festival vibe. Perhaps an initiative for 2025.
